### PR TITLE
Q35

### PR DESCRIPTION
**Pull request by VivianWenhuiLiang**

### CODE REVIEW

File: src/com/vivian/java/leetcode/Q35_SearchInsertPosition.java [hunk 1]
```diff
@@ -14,5 +14,16 @@
  * [1,3,5,6], 0 → 0 <br>
  */
 public class Q35_SearchInsertPosition {
+    public int searchInsertPosition(int[] arr, int val) {
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q35_SearchInsertPosition.java]: You approach is linear search. 
Can you write binary search?

File: src/com/vivian/java/leetcode/Q53_MaximumSubarray.java [hunk 2]
```diff
@@ -13,5 +13,28 @@
  * the divide and conquer approach, which is more subtle.
  */
 public class Q53_MaximumSubarray {
-
+    public int maximumSubarray(int[] arr) {
```
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q53_MaximumSubarray.java]: You are supposed to return back the sub array, not the sum.

File: src/com/vivian/java/leetcode/Q35_SearchInsertPosition.java (hunk 1 above)
Comment (VivianWenhuiLiang) [inline src/com/vivian/java/leetcode/Q35_SearchInsertPosition.java]: Done

File: src/com/vivian/java/leetcode/Q53_MaximumSubarray.java (hunk 2 above)
Comment (GeorgeZhiXu) [inline src/com/vivian/java/leetcode/Q53_MaximumSubarray.java]: My bad, the question asks you to return back the max sum. So you are good.

**Pull request merged**